### PR TITLE
Add chromium version 105.0.5195.102

### PR DESCRIPTION
**Pull request by timja**
Required for https://github.com/jenkins-infra/account-app/pull/167

see comments in code for some of the choices.

Unsure if it's worth adding to Windows, would rather not add it if it would never be used there

### CODE REVIEW

**Comment by timja** [comment]
It's green now @jenkins-infra/core, not reporting back the last one to GitHub for some reason
image

**Comment by timja** [comment]
possible for a review please @jenkins-infra/core?

**Comment by dduportal** [comment]
> Required for https://github.com/jenkins-infra/account-app/pull/167

Side question: since Required for https://github.com/jenkins-infra/account-app/pull/167 uses VM, can you achieve what you want with a Docker image (temporarly)?

**Comment by timja** [comment]
> (first superficial review: missing fixed version + updatecli, and missing on windows)

This may be possible with Chromium, it wasn't possible with Google Chrome as they remove old versions from the repository forcing people to stay up to date.

> missing on windows

Intentionally not added as no use-case for it, but if required can do so

> Side question: since Required for https://github.com/jenkins-infra/account-app/pull/167 uses VM, can you achieve what you want with a Docker image (temporarly)?

Could do but just complicates it and would rather keep it simpler with this.
I'm not absolutely blocked as I've got more tests to write, and can run them locally for now / disable them on CI.

**Comment by dduportal** [comment]
> > (first superficial review: missing fixed version + updatecli, and missing on windows)
> 
> This may be possible with Chromium, it wasn't possible with Google Chrome as they remove old versions from the repository forcing people to stay up to date.
> 
> > missing on windows
> 
> Intentionally not added as no use-case for it, but if required can do so
> 
> > Side question: since Required for [jenkins-infra/account-app#167](https://github.com/jenkins-infra/account-app/pull/167) uses VM, can you achieve what you want with a Docker image (temporarly)?
> 
> Could do but just complicates it and would rather keep it simpler with this. I'm not absolutely blocked as I've got more tests to write, and can run them locally for now / disable them on CI.

We'll try our best but the work on account-app is not priority for infra so we need a bit of time for this. We'll try our best to unblock you here.

**Comment by dduportal** [comment]
> This may be possible with Chromium, it wasn't possible with Google Chrome as they remove old versions from the repository forcing people to stay up to date.

Yep, we have the same issue with Git in the ubuntu PPA. At least it forces us to keep it up to date 😅 
If you can specify a fixed version, I'll take care of the updatecli manifest in an upcoming PR: looks good to you?

=> The goal is to have an auditable changelog for users (which reminds me that Docker should also be the same) because when there is a breaking change in dev tools, it is hard to audit today if version are not specified.

>>    missing on windows
> Intentionally not added as no use-case for it, but if required can do so

=> The idea is to get the same features on both OS. 

WDYT about using Chocolatey: https://community.chocolatey.org/packages?q=chromium (there is also chromium-stable)? It supports version (look at the other tools such as vagrant or ruby).
I'll also take care of the updatecli manifest for this one.

**Comment by halkeye** [comment]
you could request a saucelabs open source license, and then not really worry about keeping dependencies for chrome.

**Review by dduportal — APPROVED** [review]

**Comment by timja** [comment]
> you could request a saucelabs open source license, and then not really worry about keeping dependencies for chrome.

could do but I don't think we could run it on ci.jenkins.io which defeats a lot of the point.

File: provisioning/tools-versions.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 asdf_version: "0.10.2"
 azurecli_version: "2.41.0"
 chocolatey_version: "1.2.0"
+chromium_version: "105.0.5195.102"
```
Comment (timja) [inline provisioning/tools-versions.yml:4]: one point on this, the linux and windows available versions are currently different, windows is one version newer (although windows has old packages so it works fine).

something to keep in mind for updatecli.

could also just split them to windows / linux versions like in git

**Comment by timja** [comment]
Thanks!

**Comment by halkeye** [comment]
Sauce supports (unless they removed it) a limited time signed jwt token so it can be used in a public environment, and if the token leaks it's only good for like an hour and can only run tests not account APIs.

Not saying you should do it, just that it's an option

**Pull request merged**